examples/multicluster/usecases/floating_nse_composition/README.md

```diff
@@ -0,0 +1,113 @@
+# Floating Intedomain NSE Composition Example
+
+This example demonstrates a more complex Network Service, where we chain three passthrough and one ACL Filtering NS endpoints.
+
+The chain is formed from NSEs which are located on different clusters. It involves a combination of local and remote mechanisms.
+
+All NSEs are registered in the floating registry.
+
+```mermaid
+flowchart LR
+nsmgr-proxy1 --> nsmgr-proxy2
+subgraph cluster1
+nsc --> nsmgr--> forwarder --> nse-firewall -->| ..... | passthrough-1 -->| ..... | nsmgr-proxy1[nsmgr-proxy]
+    end
+subgraph cluster2
+    nsmgr-proxy2[nsmgr-proxy] -->| ..... | passthrough-2 -->| ..... | passthrough-3 -->| ..... | nse-kernel 
+end
+```
+Interdomain NSE Composition Scheme
+
+## Requires
+
+Make sure that you have completed steps from [interdomain](../../)
+
+## Run
+
+**1. Deploy network service on cluster3**
+
+Deploy NS:
+```bash
+kubectl --kubeconfig=$KUBECONFIG3 apply -k https://github.com/networkservicemesh/deployments-k8s/examples/multicluster/usecases/floating_nse_composition/cluster3?ref=777c5a656769d83eb31b746b4e77048aac9eecd3
+```
+
+**2. Deploy endpoints on cluster2**
+
+Deploy NSE:
+```bash
+kubectl --kubeconfig=$KUBECONFIG2 apply -k https://github.com/networkservicemesh/deployments-k8s/examples/multicluster/usecases/floating_nse_composition/cluster2?ref=777c5a656769d83eb31b746b4e77048aac9eecd3
+```
+
+Wait for applications ready:
+```bash
+kubectl --kubeconfig=$KUBECONFIG2 wait --for=condition=ready --timeout=1m pod -l app=nse-kernel -n ns-nse-composition
+```
+
+Find NSE pod by labels:
+```bash
+NSE=$(kubectl --kubeconfig=$KUBECONFIG2 get pods -l app=nse-kernel -n ns-nse-composition --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+[[ ! -z $NSE ]]
+```
+
+**3. Deploy client on cluster1**
+
+Deploy NSC:
+```bash
+kubectl --kubeconfig=$KUBECONFIG1 apply -k https://github.com/networkservicemesh/deployments-k8s/examples/multicluster/usecases/floating_nse_composition/cluster1?ref=777c5a656769d83eb31b746b4e77048aac9eecd3
+```
+
+Wait for applications ready:
+```bash
+kubectl --kubeconfig=$KUBECONFIG1 wait --for=condition=ready --timeout=5m pod -l app=alpine -n ns-nse-composition
+```
+
+Find NSC pod by labels:
+```bash
+NSC=$(kubectl --kubeconfig=$KUBECONFIG1 get pods -l app=alpine -n ns-nse-composition --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+[[ ! -z $NSC ]]
+```
+
+**4. Check connectivity**
+
+Ping from NSC to NSE:
+```bash
+kubectl --kubeconfig=$KUBECONFIG1 exec ${NSC} -n ns-nse-composition -- ping -c 4 172.16.1.100
```

Comment:
Could you use `... exec pods/alpine ...` instead of ${NSC}? 
We've simplified it a little recently. Please look at other examples